### PR TITLE
Added Git URL to GitEnricher.

### LINKED ISSUE #788: Add Git URL to GitEnricher
e.g. in 
http://central.maven.org/maven2/io/fabric8/devops/packages/logging/2.2.311/logging-2.2.311-kubernetes.yml

we just have:
```yaml
      fabric8.io/git-commit: 2b9f7ce19493fe6d18392ac4f4be16f5e1a76fa3
      fabric8.io/git-branch: release-v2.2.311
```

It would be good to include the git url too. Not sure if this is an fmp issue or something related to how we use it inside the [jenkins pipelines library](https://github.com/fabric8io/fabric8-pipeline-library/)

### PR DESCRIPTION
**Pull request by dev-gaur**
fixes #788 .

### CODE REVIEW

File: enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/GitEnricher.java [hunk 1]
```diff
@@ -61,6 +61,11 @@ public GitEnricher(MavenEnricherContext buildContext) {
                         if (id != null) {
                             annotations.put(Fabric8Annotations.GIT_COMMIT.value(), id);
                         }
+
+                        String gitOriginUrl = repository.getConfig().getString("remote", "origin", "url");
```
Comment (lordofthejars) [inline enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/GitEnricher.java]: should we offer the option to allow the user to set this as a parameter?

Comment (dev-gaur) [reply enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/GitEnricher.java]: Check now :)

File: plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java [hunk 2]
```diff
@@ -746,6 +751,7 @@ private KubernetesListBuilder generateAppResources(List<ImageConfiguration> imag
 
             // Enrich descriptors
             enricherManager.enrich(builder);
+            enricherManager.enrich(builder);
```
Comment (dev-gaur) [inline plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java]: I added this by mistakenly. #1493 fixes this. apologies.

Comment (rohanKanojia) [reply plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java]: No need to apologize, we also missed it ;-)

**Pull request merged**